pkg/minikube/cluster/cluster.go

```diff
@@ -223,11 +224,34 @@ var assets = []fileToCopy{
 	},
 }
 
-func UpdateCluster(d drivers.Driver) error {
+func UpdateCluster(h sshAble, d drivers.Driver, config KubernetesConfig) error {
 	client, err := sshutil.NewSSHClient(d)
 	if err != nil {
 		return err
 	}
+	if localkubeURLWasSpecified(config) {
+		resp, err := http.Get(util.GetLocalkubeDownloadURL(config.KubernetesVersion,
+			constants.LocalkubeLinuxFilename))
+		if err != nil {
+			return err
+		}
+		if err := sshutil.Transfer(resp.Body, int(resp.ContentLength), "/usr/local/bin",
```

Comment:
Right now I do not have a recovery plan for interrupted downloads, I will put a retry loop around the .Get that checks the err.  Also I did not store the versioned-localkube file as I stream it over ssh.  I could instead download the file and place it in 'out/' or another dir and then check whether the file (with some kind of version naming) has already been downloaded before